Cargo.toml

```diff
@@ -18,8 +18,8 @@ travis-ci = { repository = "rust-lang/libc" }
 appveyor = { repository = "rust-lang-libs/libc" }
 
 [features]
-default = ["use_std"]
```

Comment:
This was intentionally set up like this before (enabling std as a feaeture), could this be reverted?

Reply:
Sure. It's a bit more awkward because this setup allows `stdbuild` to depend on `no_std`, which can't be done in reverse, that's all.

Reply:
Done.